models/topic-model.js

```diff
@@ -65,6 +65,30 @@ const fetchCommentsByArticle = (article_id) => {
     });
 };
 
+const fetchPostCommentByArticle = (username, body, article_ID) =>{
+  if(!username || !body){
+    return Promise.reject({ status: 400, message: "Please enter a username and body" })
+  } else {
 
 
-module.exports = { fetchAllTopics, fetchArticleByID, fetchAllArticles, fetchCommentsByArticle};
+return db.query("SELECT username FROM users WHERE username = $1;", [username]).then((results)=>{
```

Comment:
I'd like to see this as a function all of its own, a checkExists util function should cut down trying to do two queries in one. That way it could be reusable.